src/Services/Bitcoin/Rpc.ts

```diff
@@ -71,9 +71,6 @@ export async function rpc<R>(method: string, args: any[] = []): Promise<R> {
     throw new BadRequestError(error.message, { code: error.code, method });
   } finally {
     const time = performance.now() - ts;
-    if (time / 1000 > 1) {
```

Comment:
should we leave this here so we are aware of slower rpc calls?

Reply:
I'll keep it but set to a higher number. RPC is slow we need to assume this, I'll set a number for extra slow RPC calls